modules/lang-painless/build.gradle

```diff
@@ -25,6 +25,7 @@ esplugin {
 }
 
 integTestCluster {
+  systemProperty 'es.script.null_for_missing_value', 'true'
```

Comment:
We should have another integ test where this value is left out, to ensure the old behavior still works.

Reply:
@rjernst  Thanks for the feedback.  What is a proper way to create another integ test?

I have got the following code, but I am not sure how to add module `mapper-extras` to `additionalClusterTestCluster`  as well.
```
integTestCluster {
  module project.project(':modules:mapper-extras')
}

Task additionalClusterTest = tasks.create(
        name: "additionalClusterTest", type: RestIntegTestTask)
additionalClusterTestCluster {
  systemProperty 'es.script.null_for_missing_value', 'true'
}
```

Reply:
>  I am not sure how to add module mapper-extras to additionalClusterTestCluster

There is not need to add it, since rest integ tests run against the entire distribution (ie it will have all modules).

Reply:
However, it would be better to do it. So, use this for the cluster config:

```
systemProperty 'es.script.null_for_missing_value', 'true'
distribution = 'integ-test-zip'
module project.project(':modules:mapper-extras')
```

Reply:
@rjernst thanks a lot for the suggestion